--- origsrc/screenruler/ruler_window.glade	2011-11-15 07:16:38.000000000 -0600
+++ src/screenruler/ruler_window.glade	2012-08-23 15:10:54.750924000 -0500
@@ -11,7 +11,6 @@
     <property name="default_height">36</property>
     <property name="type_hint">dialog</property>
     <property name="decorated">False</property>
-    <property name="has_resize_grip">False</property>
     <signal name="button-press-event" handler="on_button_press_event" swapped="no"/>
     <signal name="key-press-event" handler="on_key_press_event" swapped="no"/>
     <signal name="delete-event" handler="on_delete_event" swapped="no"/>
